package.json

```diff
@@ -56,12 +56,12 @@
     "memoizee": "0.4.1",
     "promise-worker": "^1.1.1",
     "prop-types": "^15.5.10",
-    "react": "^15.4.0",
+    "react": "^15.6.2",
     "react-addons-perf": "^15.4.0",
```

Comment:
This leaves in place [react-addons-perf](https://mvnrepository.com/artifact/org.webjars.npm/react-addons-perf) which is under BSD+patents. Should it not be removed?

Reply:
@gautampriya you're correct, this was addressed in https://github.com/swagger-api/swagger-ui/pull/5229